Tests/SDWebImage Tests.xcodeproj/project.pbxproj

```diff
@@ -568,13 +568,17 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 328BAF282240C08E00FC70DD /* Test-Debug.xcconfig */;
 			buildSettings = {
+				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
```

Comment:
we should move those settings to the xcconfig files

Reply:
This cause the warning. Because the test scheme, by default assume the version is 10.6...